storage/range.go

```diff
@@ -964,24 +956,28 @@ func (r *Range) maybeGossipConfigs(match func(configPrefix proto.Key) bool) {
 }
 
 // loadConfigMap scans the config entries under keyPrefix and
-// instantiates/returns a config map. Prefix configuration maps
-// include accounting, permissions, and zones.
-func (r *Range) loadConfigMap(keyPrefix proto.Key, configI interface{}) (PrefixConfigMap, error) {
+// instantiates/returns a config map and its checksum. Prefix
+// configuration maps include accounting, permissions, and zones.
+func (r *Range) loadConfigMap(keyPrefix proto.Key, configI interface{}) (PrefixConfigMap, uint32, error) {
 	kvs, err := engine.MVCCScan(r.rm.Engine(), keyPrefix, keyPrefix.PrefixEnd(), 0, proto.MaxTimestamp, true, nil)
 	if err != nil {
-		return nil, err
+		return nil, 0, err
 	}
 	var configs []*PrefixConfig
+	c := encoding.NewCRC32Checksum([]byte(nil))
```

Comment:
CRC32 is pretty weak and I'm worried about false matches here causing config changes to fail to propagate. This is far from performance-critical so just use a strong hash like SHA256.